### PR TITLE
Presentation: OpenTelemetry

### PR DESCRIPTION
**Pull request by Alina657**
#3796
Applied changes from [4096](https://github.com/iTwin/itwinjs-core/pull/4096) to master branch.

### CODE REVIEW

File: presentation/backend/src/presentation-backend/PresentationManager.ts [hunk 1]
```diff
@@ -296,6 +296,12 @@ export interface PresentationManagerProps {
 
   /** @internal */
   addon?: NativePlatformDefinition;
+
+  /*
+   * A function that will be called after receiving diagnostics.
+   * @alpha
```
Comment (pmconne) [inline presentation/backend/src/presentation-backend/PresentationManager.ts:302]: Why is it `@alpha`? Such APIs are not publicly documented. If you really want it to be alpha, don't include in public `NextVersion.md` documentation.

Comment (pmconne) [reply presentation/backend/src/presentation-backend/PresentationManager.ts]: Do  you intend to leave other APIs as alpha?

**Review by grigasp — APPROVED** [review]

File: presentation/backend/src/presentation-backend/PresentationManager.ts (hunk 1 above)
Comment (Alina657) [reply presentation/backend/src/presentation-backend/PresentationManager.ts]: Changed all APIs related to diagnostics to `@beta`.

**Review by pmconne — COMMENTED** [review]
Beta APIs appear in documentation. Some of yours lack documentation.

**Comment by Alina657** [comment]
> Beta APIs appear in documentation. Some of yours lack documentation.

@pmconne added documentation.

**Review by pmconne — APPROVED** [review]
Documentation needs improvement before this can be promoted to public.

**Pull request merged**